server/models/Document.test.js

```diff
@@ -430,3 +430,79 @@ describe("#findByPk", () => {
     expect(response.id).toBe(document.id);
   });
 });
+
+describe("tasks", () => {
+  test("should consider all the possible checkTtems", async () => {
+    const document = await buildDocument({
+      text: `- [x] test
+      - [X] test
+      - [ ] test
+      - [-] test
+      - [_] test`,
```

Comment:
They are matched through the regex, but I am not sure about their behaviors. Are they considered checked or unchecked?

Reply:
✅  checked

Reply:
My guess was right 😄